scripts/tests.sh

```diff
@@ -43,7 +43,8 @@ test "${output//[[:blank:]]/}" = "3"
 if [[ "$OSTYPE" == "darwin"* ]]; then
     ETH_PATH="$REPO_ROOT/eth"
 else
-    ETH_PATH="eth"
```

Comment:
The fact that mac is that way is a problem that we need to solve. It should have been gotten in the install deps script and should be in the path imo.